src/components/multi-pages-modal/MultiPagesModal.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import { MultiPagesModalProps } from './types';
+import { Modal, ModalBody, ModalFooter, ModalHeader, ModalSize } from '../modal';
+import { Button } from '../button';
+import { ButtonKind, ButtonType } from '../../models';
+
+export const MultiPagesModal = ({
+  isOpen,
+  onClose,
+  loading,
+  title,
+  pages,
+  submitText,
+  resetOnClose,
+  disableCounterTitle = false,
+  nextPageDisabled = false,
+  noSubmit = false,
+  onSubmit,
+  validatePage,
+  cancelText,
+  previousText,
+  nextText,
```

Comment:
Maybe we can add name as optional property so you are able to pass on an unique name. I don't think we have multiPagesModal inside multiPagesModal ( as we have for regular modals ), but if we want that in the future we will need unique names to keep track which ones are open. 
```suggestion
  name = "multi-pages-modal"
```

Reply:
Yes, you are absolutely right